interfaces/slack.js

```diff
@@ -195,23 +195,29 @@ const slackInterface = {
     },
     sendChannelUpdate(update, done) {
         updateModel.get(update.user).then(update => {
-            this.webClient.chat.postMessage(
-                update.channel,
-                '',
-                generateAttachment(update),
-                (err, res) => {
-                    err
-                        ? console.log(
-                              `FAILED TO SEND CHANNEL UPDATE TO ${
-                                  update.channel
-                              } FROM ${update.user}: `,
-                              err
-                          )
-                        : console.log('Message sent: ', res);
-                    update.editStatus('ts', res.ts);
-                    done();
-                }
-            );
+            update.channel.forEach((channel, i) => {
+                setTimeout(
+                    () =>
+                        this.webClient.chat.postMessage(
+                            channel,
+                            '',
+                            generateAttachment(update),
+                            (err, res) => {
+                                err
+                                    ? console.log(
+                                          `FAILED TO SEND CHANNEL UPDATE TO ${channel} FROM ${
+                                              update.user
+                                          }: `,
+                                          err
+                                      )
+                                    : console.log('Message sent: ', res);
+                                update.editStatus('ts', res.ts);
+                                done();
+                            }
+                        ),
+                    1000 * i
```

Comment:
bit arb but ok 👍

Reply:
It's not arb - slack rate limiting is 1 msg/sec :stuck_out_tongue:

Reply:
oooh ok per user or per bot? how do you handle multiple ppl then?

Reply:
That gets managed by redis :)